spec/features/javascript_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+feature 'JavaScript Tracking', feature: true do
+  before { Raygun.configuration.js_api_key = nil }
+  after { Raygun.configuration.js_api_key = nil }
+
+  it "Includes the Raygun Javascript Middleware" do
+    expect(Rails.application.config.middleware).to include(Raygun::Middleware::JavascriptExceptionTracking)
+  end
+
+  it "No JS snippet is injected" do
```

Comment:
I would wrap this in a context 'Without JS Api key'. I'm also a fan of the `it` being included in the test name so would rephrase to something more like `it 'does not inject JS snippet'`